src/index.js

```diff
@@ -40,6 +41,10 @@ export default class ReactImageFallback extends Component {
 			}
 			this.setState({
 				imageSource: image
+			}, () => {
+				if (this.props.onLoad) {
+					this.props.onLoad(image);
+				}
```

Comment:
You need to delete the corresponding onLoad call above, otherwise you're calling twice. Otherwise, looks good!